label_studio/data_import/api.py

```diff
@@ -604,19 +604,23 @@ def get(self, request, *args, **kwargs):
 
         fileuri = unquote(fileuri)
 
-        resolved = task.resolve_storage_uri(fileuri, project)
+        try:
+            resolved = task.resolve_storage_uri(fileuri, project)
+        except Exception as exc:
+            logger.error(f'Failed to resolve storage uri {fileuri} for task {task_id}: {exc}')
+            return Response(status=status.HTTP_404_NOT_FOUND)
 
-        if resolved is None or resolved['url'] is None:
+        if resolved is None or resolved.get('url') is None:
             return Response(status=status.HTTP_404_NOT_FOUND)
 
-        url = resolved['url']
-        maxAge = 0
-        if resolved['presign_ttl']:
-            maxAge = resolved['presign_ttl'] * 60
+        url = resolved.get('url')
```

Comment:
I'm not sure we need `.get` here. We already had this check before, but if we will remove it in the future, I think it's a bit clearer exception, than redirect to None, that we'll get in this case.